.github/workflows/ci_special.yml

```diff
@@ -47,7 +47,7 @@ jobs:
     runs-on: ubuntu-22.04
     if: always()
     steps:
-      - uses: technote-space/workflow-conclusion-action@45ce8e0eb155657ab8ccf346ade734257fd196a5 # tag v3.0.3
+      - uses: ./.github/workflows/scripts//workflow-conclusion # no tag - locally sourced
```

Comment:
Doe the double `//` cause a problem? I would've expected to see `./.github/workflows/scripts/workflow-conclusion`

Reply:
Oh! This def would've cause an issue. I suppose this action isn't running since this only runs on a merge. I will fix and look into this